### PR TITLE
Added Drupal specific coding standards

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**

### CODE REVIEW

File: content/docs/how-we-work/drupal/coding.md [hunk 1]
```diff
@@ -23,16 +23,33 @@ If we have written a generic implementation, others in the project can use it ve
...
 - [Writing secure code for Drupal](https://www.drupal.org/docs/administering-a-drupal-site/security-in-drupal/writing-secure-code-for-drupal)
 
-## Coding Standards
+## Drupal-specific Coding Standards
 
-We apply the Drupal coding standards when coding in Drupal projects. We always make sure that these standards are up-to-date and version-independent.
+- We follow [Drupal Coding Standards](https://www.drupal.org/docs/develop/standards), which include PHP, CSS, JavaScript, Twig, YAML, and more.
 
-- [Drupal coding standards and best practices](https://www.drupal.org/docs/develop/standards)
+- We ensure [Code Quality with DCQ](https://www.drupal.org/project/dcq) which includes the Coder module to ensure code meets Drupal standards.
```
Comment (hussainweb) [inline content/docs/how-we-work/drupal/coding.md:38]: Does someone use this module at Axelerant? I know some of our projects use https://github.com/axelerant/drupal-quality-checker

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: content/docs/how-we-work/drupal/coding.md (hunk 1 above)
Comment (Yemaneberhan-Lemma) [reply content/docs/how-we-work/drupal/coding.md:38]: I checked it, and it was last updated in March 2023, but it works with Drupal 7.x. I'll keep [this](https://github.com/axelerant/drupal-quality-checker) in mind.

Comment (Kalaiselvan88) [reply content/docs/how-we-work/drupal/coding.md:38]: Yes. I think we can mention https://github.com/axelerant/drupal-quality-checker instead of this

Comment (Yemaneberhan-Lemma) [reply content/docs/how-we-work/drupal/coding.md]: @hussainweb and @Kalaiselvan88: Thanks. As per your suggestion, I changed it to Drupal Quality Checker.

**Review by hussainweb — APPROVED** [review]

**Pull request merged**